### PR TITLE
feat: sort husky hooks

### PR DESCRIPTION
**Pull request by fisker**
Not sure is this #20 requested, we should sort them anyway.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,6 +15,38 @@ const sortObject = onObject(sortObjectKeys())
...
+    'post-update',
+    'push-to-checkout',
+    'pre-auto-gc',
+    'post-rewrite',
+    'rebase',
+    'sendemail-validate',
+    'fsmonitor-watchman',
+    'p4-pre-submit',
+    'post-index-change',
+  ],
```
Comment (fisker) [inline index.js]: You prefer keep this list here, or a separate package I build https://github.com/fisker/git-hooks-list ?

Comment (keithamus) [reply index.js]: Yes I was thinking we could start moving a lot of this stuff out to dependencies. I'm happy to add this dependency 👍

Comment (fisker) [reply index.js]: done.

**Pull request merged**